modules/afprog/afprog.c

```diff
@@ -271,40 +298,66 @@ afprogram_dd_format_persist_name(AFProgramDestDriver *self)
   return persist_name;
 }
 
+static gchar *
+afprogram_dd_format_persist_name(AFProgramDestDriver *self)
+{
```

Comment:
This was an ugly workaround, because sometimes the queue is persisted, too, and I am afraid this is beyond my control, therefore a new persist-entry was needed.
I should check this queue persist logic before going any further.